praw/models/reddit/submission.py

```diff
@@ -601,6 +628,34 @@ def suggested_sort(self, sort="blank"):
             data={"id": self.thing.fullname, "sort": sort},
         )
 
+    def unset_original_content(self):
+        """Indicate that the submission is not original content.
+
+        This method can only be used by moderators of the subreddit
+        that the submission belongs to, if the subreddit has enabled
+        the beta feature in settings. This uses the same endpoint as
+        `set_original_content`, but `should_set_oc` is set to `False`.
```

Comment:
I believe the sentence about what endpoint the method uses should be removed, because PRAW's users shouldn't have to worry about which endpoints are being used.